Minecraft_BTE_Tips/main.css

```diff
@@ -0,0 +1,372 @@
+  border-top-left-radius: 3px;
+  border-bottom-left-radius: 3px;
+}
+
+.navigation-visibility {
+  z-index: 999;
+  display: flex;
+  justify-content: center;
+}
+
+.navigation-visibility .slide-icon {
+  z-index: 999;
+  background: rgba(255, 255, 255, 0.5);
+  width: 20px;
+  height: 10px;
+  transform: translateY(-50px);
+  margin: 0 6px;
+  border-radius: 2px;
+  box-shadow: 0 5px 25px rgb(1 1 1 / 20%);
+}
+
+.navigation-visibility .slide-icon.active {
+  background: #4285f4;
+}
+
+@media (max-width: 900px) {
+  .slider {
+    width: 100%;
+    margin-left: 3px;
+  }
+
+  .slider .slide .info {
+    position: relative;
+    width: 80%;
+  }
+}
+
+@media (max-width: 500px) {
+  .slider .slide .info h2 {
+    font-size: 1.8em;
+    line-height: 40px;
+    margin-left: 2px;
+    margin-right: 8px;
+  }
+
+  .slider .slide .info p {
+    font-size: 0.9em;
+  }
+}
+
+header {
+  background-image: url(./Assets/Images/minecraft_builds/MC_SingaporeSkyline_1.png);
+  background-repeat: none;
+  background-size: cover;
+  background-position: 50% 0%;
+  position: relative;
+}
+
+header::after {
+  display: block;
+  content: "";
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  position: absolute;
+  z-index: 1;
+  background-color: rgba(0, 0, 0, 0.75);
+}
+
+header .container {
+  position: relative;
+  z-index: 2;
+  padding: calc(var(--pt8) * 12) calc(var(--pt8) * 3);
+}
+
+header h1 {
+  font-weight: bold;
+  margin-bottom: calc(var(--pt8) * 4);
+}
+
+header h1,
+header h4 {
+  color: #ffffff;
+}
+
+header .viewmore {
+  background-color: #47bb2a !important;
+  color: #ffffff;
+  border-color: #47bb2a;
+  /* Font */
+  text-transform: uppercase;
+}
+
+header .viewmore::after {
+  background-color: #3ca123;
+}
+
+.viewmore i {
+  transition: var(--ease-out-expo);
+}
+
+header .viewmore:hover.viewmore i {
+  transform: rotate(-90deg);
+}
+
+.viewmore {
+  position: relative;
+  overflow: hidden;
+  margin-top: 1rem;
+  padding: 1rem 3rem !important;
+  background-color: #e6e6e6 !important;
+  /* Font */
+  font-size: 0.875rem !important;
+}
+
+.viewmore:hover {
+  background-color: #dddddd !important;
+}
+
+.viewmore span {
+  position: relative;
+  z-index: 3;
+}
+
+.viewmore:hover.viewmore span {
+  color: #ffffff;
+}
+
+.viewmore::after {
+  content: "";
+  width: 0;
+  height: 100%;
+  background-color: var(--bs-dark);
+  position: absolute;
+  left: 0;
+  top: 0;
+  z-index: 1;
+  /* Transition */
+  transition: var(--ease-out-expo);
+}
+
+.viewmore:hover:after {
+  width: 100%;
+}
+
+@media (max-width: 770px) {
+  header h1,
+  header h4 {
+    text-align: center;
+  }
+
+  header .viewmore {
+    width: 100%;
+  }
+
+  .landingpage_logo {
+    margin-bottom: 2.5px;
+  }
+}
+   
+/*Apple Watch (Parrity) 38mm 40mm 42mm*/
+@media (max-width: 400px) {
+  .slider .slide .info h2 {
+    font-size: 1.2em;
+    line-height: 40px;
+    margin-left: 4px;
+    margin-right: 2px;
+  }
+
+  .slider .slide .info p {
+    font-size: 0.4em;
+  }
+}
```

Comment:
Similarly this is the CSS for the `header` tag and the `viewmore` button, so there is no use of them over here